processccd.rst

```diff
@@ -1,131 +1,100 @@
 
-2/4
 
-ProcessCCD
-================
+##############
+ProcessCcdTask
+##############
 
+ProcessCcd (which is a ``Cmd Line Task``, and thus can be run directly
+by typing processCcd.py at a shell prompt) executes the steps of how
+an image is processed from raw data to a science-grade images and catalogs that can
+be used in analyses.
 
-- `Doxygen link`_
-.. _Doxygen link: https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/classlsst_1_1pipe_1_1tasks_1_1process_ccd_1_1_process_ccd_task.html#ProcessCcdTask_
+In more detail: ProcessCcd as a whole executes many of the functions
+that are in multiple packages in other astronomy analysis frameworks.
```

Comment:
Sounds good - i did leave in for now "In more detail" too, because i feel it flows better from the first sentence, and is just 3 more words.. but, if there's a consensus to remove it, that's fine too.